Library/Formula/google-java-format.rb

```diff
@@ -0,0 +1,18 @@
+class GoogleJavaFormat < Formula
+  desc "Tool to format java code with the google java format"
+  homepage "https://github.com/google/google-java-format"
+  url "https://github.com/google/google-java-format/releases/download/google-java-format-0.1-alpha/google-java-format-0.1-alpha.jar"
+  version "0.1-alpha"
```

Comment:
This needs to be a stable release (e.g. non-alpha/non-beta) to be included in Homebrew.